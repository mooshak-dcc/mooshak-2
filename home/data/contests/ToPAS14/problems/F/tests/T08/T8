515 5 9
AFFFAFFFF
AVVVVVXVA
AVVVIVXVA
AVVVVVTVF
AAAAAAAAA
2
E6
N2
