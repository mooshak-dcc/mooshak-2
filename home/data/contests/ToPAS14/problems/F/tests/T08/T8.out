Dentro(7,3):464:ND
